docs/reference/controls/itemsrepeater.md

```diff
@@ -127,7 +127,7 @@ The items display horizontally, and those too far to the right would be hidden i
 ## More Information
 
 :::info
-For the complete API documentation about this control, see [here](http://reference.avaloniaui.net/api/Avalonia.Controls/ItemsRepeater/).
+For the complete API documentation about this control, see [here](TODO).
```

Comment:
How should we deal with this? The ItemsRepeater is no longer maintained. I'd prefer to just delete this section and add a warning badge at the top. Source could be linked to new repo however.

Reply:
Yeah, let's just remove this paragraph